pkgs/games/cgoban/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib
+, stdenv
+, writers
+, fetchurl
+, adoptopenjdk-jre-bin
+, ver ? "3.5.23"
+, hash ? "0srw1hqr9prgr9dagfbh2j6p9ivaj40kdpyhs6zjkg7lhnnrrrcv" }:
+let
+  cgoban_jar = stdenv.mkDerivation rec {
```

Comment:
Why do we need this double mkDerivation? Can't we just throw them together?

Reply:
Without the double derivation i"m having a very hard time with the launcher script. `writeBash` doesn't have the path to `$out` at the time that I define it. I can investigate whether I can make a launcher script like this with `makeWrapper`.

Reply:
You can get the value of out with ``placeholder "out"``.